internal/literals/literals.go

```diff
@@ -81,63 +81,63 @@ func Obfuscate(file *ast.File, info *types.Info, fset *token.FileSet, ignoreObj
 		case *ast.CompositeLit:
 			byteType := types.Universe.Lookup("byte").Type()
 
-			if len(x.Elts) == 0 {
+			if len(x.Elts) == 0 || len(x.Elts) > maxSizeBytes {
 				return true
 			}
 
+			var arrayLen int64
 			switch y := info.TypeOf(x.Type).(type) {
 			case *types.Array:
 				if y.Elem() != byteType {
```

Comment:
this can also be shared between Array and Slice, after the switch.

Reply:
I don't think it can

Reply:
Ah, right, the type switch. I guess you could use something like `if y.(interface{Elem() types.Type).Elem() != byteType {`, but that feels overkill.